src/keymap.cpp

```diff
@@ -484,6 +484,14 @@ static const std::vector<OpDesc> opdescs = {
 	{OP_INT_SET, "set", "internal-set", "", KM_INTERNAL},
 
 	{OP_INT_GOTO_URL, "gotourl", "internal-goto-url", "", KM_INTERNAL},
+
+	{
```

Comment:
Please move this up before `OP_SK_*` and `OP_INT_*` ops.

Reply:
@Minoru maybe even before `OP_CMD*` and `OP_OPEN*`?

Reply:
Agreed, that'd look better.